Automattic/pocket-casts-android
# Transcripts - Replace transcript shelf icon

## Description
This replaces transcript shelf icon.

## Testing Instructions
1. Play an episode with Transcript (Cautionary Tales" (https://pcast.pocketcasts.net/8zpmjiru))
2. Open full-screen player 
3. ✅ Notice the transcript shelf icon is replaced with the new one (grabbed from iOS [PR](https://github.com/Automattic/pocket-casts-ios/pull/2004))
4. Open Rearrange view
5. ✅ Notice that the icon is shown properly

## Screenshots or Screencast 

Shelf | Rearrange Actions
---|---
<img width = 320 src="https://github.com/user-attachments/assets/d3e6269f-d332-408c-b971-3f7cdb30196d"/>|  <img width = 320 src="https://github.com/user-attachments/assets/759a15d7-9dd6-47f0-b6cf-6f2157c816eb"/>

Landscape

<img width = 400 src="https://github.com/user-attachments/assets/0cede41f-a342-4074-9a44-c856274fa423"/>

Tablet 
<img width = 500 src="https://github.com/user-attachments/assets/bdf7e301-fce4-40ba-9c49-f57476c08a0a"/>


## Checklist
 
#### I have tested any UI changes...
